src/postprocessor.rs

```diff
@@ -244,6 +246,25 @@ impl Postprocessor for LengthPrefix {
     }
 }
 
+#[derive(Clone, Default)]
+pub(crate) struct TextualLength {}
+impl Postprocessor for TextualLength {
+    #[cfg(not(tarpaulin_include))]
+    fn name(&self) -> &str {
+        "textual-length"
+    }
+
+    fn process(&mut self, _ingres_ns: u64, _egress_ns: u64, data: &[u8]) -> Result<Vec<Vec<u8>>> {
+        let mut res = Vec::new();
```

Comment:
please allocate a new array with the known length. with something like: `digits.len() + 1 + data.len()`. This will be faster and avoid copying stuff around when the Vec needs to be resized.